packages/webpack-cli/lib/bootstrap.js

```diff
@@ -23,6 +24,11 @@ const runCLI = async (cliArgs) => {
         // If the unknown arg starts with a '-', it will be considered an unknown flag rather than an entry
         let entry;
 
+        // enable/disable colors
+        if (parsedArgs.opts.hasOwnProperty('color')) {
```

Comment:
Please fix CI problem

Reply:
Why do you need `hasOwnProperty`? we don't use prototypes here, so just use `parsedArgs.opts.color`

Reply:
https://github.com/webpack/webpack-cli/pull/2042#discussion_r519973255

Reply:
```suggestion
        if (typeof parsedArgs.opts.color !== 'undefined') {
```

Reply:
strange that problems occur with hasOwnProperty, let's go with this for now